tests/app/main/test_services.py

```diff
@@ -155,7 +155,8 @@ def test_shows_dos_service_name_without_edit_link(self, data_api_client):
                     'lotName': 'Special Lot Name',
                     'status': 'published',
                     'id': '123',
-                    'frameworkSlug': 'digital-outcomes-and-specialists'
+                    'frameworkSlug': 'digital-outcomes-and-specialists',
```

Comment:
If it is a one line change I might advocate that `frameworkSlug` should be for DOS2. That way we know that our code is using `frameworkFramework` instead or the same valued `frameworkSlug` at the moment.

Or maybe I'm being anal....

👍 either way